### PR TITLE
Add basic Google Maps & Hide API key for maps

### PR DESCRIPTION
**Pull request by KathleenQ**
This PR contains the basic function to add Google Maps to the website. Some related format changes are also included.
It has been updated from the closed PR #6 to hide the API key in a json file and avoid to add the key onto GitHub for security. 
(The previous API key has now been deleted and a new key has been generated privately for use.)

### CODE REVIEW

**Review by tripd22 — COMMENTED** [review]
Could you please update the description of this PR? I know it was originally hiding the API key for maps, but now it is the PR that actually adds the maps functionality.

**Comment by lilian-hunt** [comment]
lgtm

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -53,3 +53,28 @@ function fetchBlobstoreUrlAndShowForm() {
...
+      script.defer = true;
+      window.initMap = function() {
+        createBasicMap();
+      };
+      document.head.appendChild(script);
+    });
+
+/** Creates the basic map. */
+function createBasicMap() {
+  map = new google.maps.Map(document.getElementById('map'), {
```
Comment (aannie-y) [inline portfolio/src/main/webapp/script.js]: As Robert mentioned in my PR, maybe have a constant or some way explaining the location.

**Review by aannie-y — COMMENTED** [review]
Other than the map const, lgtm!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (KathleenQ) [reply portfolio/src/main/webapp/script.js]: Yes, defining the map constant makes sense! I have just updated that. I think that is what you mean, right?

Comment (tripd22) [reply portfolio/src/main/webapp/script.js]: You could use the variable name to describe the location [like Annie did](https://github.com/aannie-y/step/pull/14/files).

Comment (KathleenQ) [reply portfolio/src/main/webapp/script.js]: Sure. I have just added the location in the map name (as description) as well.

**Review by aannie-y — COMMENTED** [review]
lgtm

**Review by tripd22 — APPROVED** [review]

**Pull request merged**